tests/unit/test_config_logging.py

```diff
@@ -39,3 +43,23 @@ def test_configure_logging_mozlog_production():
         assert "Log format must be 'mozlog' in production" in str(excinfo)
 
         settings.logging.format = old_format
+
+
+def test_log_handler_assigned_mozlog() -> None:
+    """Test that the log handler is assigned as expected when the configured format is 'mozlog'"""
+    settings.logging.format = "mozlog"
+    configure_logging()
+
+    merino_log_manager: Any = logging.root.manager
+    handler_name = merino_log_manager.loggerDict["merino"].handlers[0].name
+    assert handler_name == "console-mozlog"
```

Comment:
If you'd like to specify 'Manager' instead of 'Any', you can do the following, but I'm not really sure if that's better...

```suggestion
    merino_log_manager: Manager = logging.root.manager
    merino_logger: Logger | PlaceHolder = merino_log_manager.loggerDict["merino"]
    assert isinstance(merino_logger, Logger)
    assert merino_logger.handlers[0].name == "console-mozlog"
```
Note: Logger, Manager & Placeholder need to be imported from `logging`

Reply:
Thanks so much for the review @Trinaa ! I'll look over and apply the changes. Good recs

Reply:
Yeah, as for this, MyPy isn't smart enough to pick up `PlaceHolder` and `Logger`. Even though the code works, it gets mad as it things there is no `handlers` attribute. I mucked around with that a while trying to get it to work, so I think let's leave it as `Any` if you're comfortable. It'd be nice to get it right, but I don't think it'll work with MyPy. Plus, not as important since we at least have the `Manager` type correctly bound.